__tests__/app.test.js

```diff
@@ -227,5 +227,67 @@ describe("POST /api/articles/:article_id/comments", () => {
    });
  });
 });
-
-//testing
+describe("/api/articles/:article_id", () => {
+ test("should return the correct status code and the updated article object", () => {
+  const testObject = { inc_votes: 20 };
+  return request(app)
+   .patch("/api/articles/3")
+   .send(testObject)
+   .expect(201)
+   .then(({ body }) => {
+    expect(body.article).toEqual(
+     expect.objectContaining({
+      article_id: 3,
+      title: "Eight pug gifs that remind me of mitch",
+      topic: "mitch",
+      author: "icellusedkars",
+      body: "some gifs",
+      created_at: expect.any(String),
+      votes: 20,
```

Comment:
this test would be better if you checked what the votes were before the patch request and then after. While it's good to check that all the other properties on the object are still there, I think it's maybe not needed if we're just concerned about the votes